vcopy.data

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Did you mean to add these two files (`vcopy.data` and `vcopy.futil`)?

Reply:
not to master, I'll remove them